x-pack/plugins/apm/public/components/app/service_inventory/service_groups/service_group_card.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import {
+  EuiAvatar,
+  EuiCard,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import {
+  ServiceGroup,
+  SERVICE_GROUP_COLOR_DEFAULT,
+} from '../../../../../common/service_groups';
+
+interface Props {
+  serviceGroup: ServiceGroup;
+  onClick: () => void;
+  showServiceCount?: boolean;
+}
+
+export function ServiceGroupsCard({
+  serviceGroup,
+  onClick,
+  showServiceCount = true,
```

Comment:
nit: If this is enabled by default, we should flip the setting:

```suggestion
  hideServiceCount = false,
```

This way the consumer will do `<ServiceGroupsCard  hideServiceCount />` instead of `<ServiceGroupsCard  showServiceCount={false} />`